Sofa/framework/Core/src/sofa/core/ConstraintParams.h

```diff
@@ -35,14 +35,40 @@ class SOFA_CORE_API ConstraintParams : public sofa::core::ExecParams
 public:
 
     /// Description of the order of the constraint
-    enum ConstOrder
+    enum class ConstOrder
```

Comment:
Food for thought:
I feel a bit confused by this `ConstOrder` ; on a first glance, I read "Constant Order" instead of "Constraint Order".
Especially that a bit further, we have an other "Const" with `ConstMultiVecCoordId` with the "Constant" meaning in this case.